docs/topics/braintree.rst

```diff
@@ -227,6 +227,16 @@ solitude.
 
     :status 201: payment method created.
 
+Change payment method on a subscription:
+
+.. http:post:: /braintree/subscription/change/
```

Comment:
If you are still opposed to PUT, another thing that might make this more intuitive is to put the ID in the resource, like: ``/braintree/subscription/123/change/`` because otherwise it seems odd to me that you'd have to POST the ID to the subscription.